controllers/cryptoController.js

```diff
@@ -0,0 +1,96 @@
+const usersdata = require('../models/crypto')
+const calculation = require('../middlewares/cryptoTransactionCalc')
+/**
+ * Send Money from one User to Other
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ * {
+ *  "to": "kratika",
+ *  "from": "uttam",
+ *  "amount": 25,
+ *  "currency": "dinero"
+ * }
+ */
+const send = async (req, res) => {
+  try {
+    const request = req.body
+    const [userTo, userFrom, currency, amount] = [request.to, request.from, request.currency, request.amount]
+    const message1 = `${amount} ${currency} Coins Creditted From ${userFrom} to ${userTo} Succesfully`
+    const [fromUserWallet, toUserWallet] = await Promise.all([usersdata.fetchUserWallet(userFrom), usersdata.fetchUserWallet(userTo)])
```

Comment:
Do we really need to get receiver wallet with sender. Think about dependency here.